docs/articles/user-guide/org-browser.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Org Browser
+---
+
+The Org Browser feature allows you to view the available metadata types and their corresponding components in the default org. It is another option for retrieving metadata source without having to use a [manifest file](./org-development-model#the-manifest-packagexml-file). The Org Browser is only available when working against non-scratch orgs, such as a production, sandbox, or devhub org.
```

Comment:
I would modify this to `against non-scratch orgs, such as sandboxes or dev orgs.`. Ideally they shouldn't be coding against a production org and that's what the Org Browser would facilitate.